app/graphql/mutations/movie_mutation.rb

```diff
@@ -0,0 +1,15 @@
+module Mutations
+    class MovieMutation < BaseMutation
```

Comment:
Consider naming convention - CreateMovieMutation, cause you will have separated Update and Create Mutations

Reply:
Ok.